modules/standard/Path.chpl

```diff
@@ -189,4 +189,40 @@ proc file.realPath(): string {
    }
  }
 
+ /*
+
+ Returns the parent directory of the :type:`~IO.file` record.  For instance,
+ a file with path `/foo/bar/baz` would return `/foo/bar`
+
+  :return: The parent directory of the file
+  :rtype: `string`
+
+ */
+ proc file.getParentName(out error:syserr): string {
```

Comment:
Almost perfect!  Would you throw a `pragma "no doc"` on this one, and remove the tmp panzone pointed out?  I will merge this once that happens :)